### DISCUSSION TITLE
[Custom]: getting custom eww and ags bars to open with different themes

### DISCUSSION THREAD

**Discussion by Itstatertots** [prasanthrangan/hyprdots#1042] [Q&A]
i prefer eww and ags and was wondering if there was a way to say get a ags bar to open with the gruvbox theme then if i changed to the rose pine theme a eww bar opens instead

**Comment by kRHYME7** [comment]
@Itstatertots Sorry, I migrated this to discussion panel to have more space for discussion. 


You can add custom script to execute. The same goes to other themes.  So when you change themes it will be executed. 
Example   ``` ~/.config/hypr/themes/Gruvbox-Retro.conf ``` are the hyprland configuration for gruvbox you can execute your custom script in here .   Same goes to the other themes. (Note that theme.conf is a symlink of the current theme)

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/5e58041e-e28e-4265-8268-06731023caf9)



#### TIps!

The themes are determined using this command ```  gsettings get org.gnome.desktop.interface gtk-theme | sed "s/'//g" ``` 
e.g
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/403aacda-cae0-4574-9dd4-b9ea72536abb)



BTW, can you share some of your juicy ags and eww configs?

**Reply by Itstatertots** [reply]
Thank you so much this worked perfectly. As for sharing my eww and ags configs I'm still learning yuck so I'm not confident in my bars yet but when I am I'll upload them

**Comment by noahfraiture** [comment]
I saw a post about ags and eww. Are there any plan to switch to one of these?

**Reply by Oskar-Idland** [reply]
Eww bars and widgets are a lot better than waybar ngl.
